scripts/transition_metrics.py

```diff
@@ -29,6 +29,35 @@ def get_metrics(filename):
     return evaler
 
 
+def add_zeros_columns(df, column_names):
+    '''
+    Adds a column of a specified name to a given dataframe
+    if a column of that name does not exist already. The
+    added column is of the length of the entire dataframe
+    but consists of only zeros. This function allows for
+    greater flexibility in defining prototypes of
+    interest across multiple tranistion scenarios
+
+    Parameters:
+    -----------
+    df: DataFrame
+        dataframe to add column to, if the column doesn't exist
+        already
+    column_names: list of strs
+        names to be checked for existence and added if missing
+
+    Returns:
+    --------
+    df: DataFrame
+        dataframe with added column, if column doesn't
+        exist anymore
+    '''
+    for item in column_names:
+        if item not in df.columns:
+            df[item] = 0.0
+    return df
+
+
 def rx_commission_decommission(filename, non_lwr):
```

Comment:
This function name is a bit unclear. Consider changing it to better elucidate its purpose.